Wikipedia/Code/NotificationsCenterCell.swift

```diff
@@ -66,10 +68,17 @@ final class NotificationsCenterCell: UICollectionViewCell {
         label.textAlignment = .left
         label.numberOfLines = 1
         label.setContentCompressionResistancePriority(.defaultLow, for: .horizontal)
-        label.setContentHuggingPriority(.required, for: .horizontal)
+        label.setContentHuggingPriority(.defaultLow, for: .horizontal)
```

Comment:
@staykids My only thinking with this particular change was to let the `headerLabel` expand so that it's easier for the user to tap, rather than having `relativeTimeAgoLabel` be the one that takes up most of the container view. I can revert this though if you think it's not necessary or causes a layout regression.

The other constraint tweaks in this file were necessary for the tap gesture on `headerLabel` to fire. After inspecting the view hierarchy,  I noticed the `headerTextContainer`'s width was evaluating to zero (even though it's subviews were displaying fine!), which I think caused the tap gesture recognizer on the headerLabel to be ignored. Hopefully these other tweaks didn't cause any layout regressions. Let me know if you see any!

Reply:
Gotcha – thanks for the details! I haven't yet noticed a layout regression with the change.